src/lib/graphql/queries/getVariableEntity.query.ts

```diff
@@ -0,0 +1,39 @@
+import { gql } from '@apollo/client';
+
+export const GET_VARIABLE_ENTITY = gql`
+  query getVariables(
```

Comment:
getVariables to getVariablesEntity car getVariables existe deja

Reply:
ok